src/nullablevector.jl

```diff
@@ -1,3 +1,5 @@
+import Base: hcat, vcat
```

Comment:
This whole file uses `Base.` to prefix everything. Should I add all of the functions to this import call? It's an unrelated change but importing these two and leaving everything else to be `Base.` is inconsistent